generator/sbpg/targets/latex.py

```diff
@@ -80,7 +80,7 @@ def nobrackets(v):
   """
   Remove empty brackets
   """
-  return v.translate(None, '[]')
+  return v.replace('[', '').replace(']', '')
```

Comment:
Doesn't this replace even if the brackets are not empty?

Reply:
it does but so does the original. So there is either an existing bug or the description is bad. Good point anyway

Reply:
I studied this a bit more. Short story: the description was bad (changed to "Remove brackets"). 

Longer story: This function is used for creating cross-reference labels for bitfield documentation so the output is only indirectly visible, and the exact return value of this function doesn't actually matter. At least on my machine it also worked even without removing the brackets, i.e, with references containing things like `blaah[N]`, so it looks like an unnecessary hack to me, but I guess there could be some unknown-to-me reason for it. I left it as it was for now. If somebody requests, I can do the removal in a separate PR.